src/Workspaces/CSharp/Portable/Extensions/SemanticModelExtensions.cs

```diff
@@ -445,5 +477,103 @@ private static TypeSyntax GetOutermostType(TypeSyntax type)
         {
             return type.GetAncestorsOrThis<TypeSyntax>().Last();
         }
+
+
+        // API that  gets semantic info for a specific arguments from @dustinca
+        //https://github.com/DustinCampbell/CSharpEssentials/blob/master/Source/CSharpEssentials/Extensions.cs
+        // supposedly there is a bug on github somewhere for him to add this api to the compiler
+        internal struct ArgumentInfo
```

Comment:
Based on my comment below, this should probably be public to make the accessibiliity of the API that returns these. :smile: